docs/source/implementations/backtest_overfitting.rst

```diff
@@ -0,0 +1,146 @@
+.. _implementations-backtest_overfitting:
+
+============================================================
+Backtest Overfitting by Campbell R. Harvey and Yan Liu
```

Comment:
The original paper is just called ```Backtesting``` Lets remove Overfitting from the title. Unless I missed something?

Reply:
It is. I was still thinking about the naming. Will change.